SRC/Pages/OVWObjectViewer.Page.al

```diff
@@ -194,4 +210,15 @@ page 50130 "OVW Object Viewer"
             exit;
         exit(OVWRecordCountMgt.GetRecordCount(ForObjectID));
     end;
+
+    local procedure DoExportToExcel()
```

Comment:
This is a good data validation, but you might also add some code on the action in the page extension so that it is only enabled when the object type is Table.